### PR TITLE
Don't use sudo on delegate task

### PR DESCRIPTION
**Pull request by cooktheryan**
Using sudo on delegate tasks causes priv escalation on the system running ansible or on ansible auto hub. We should not need sudo for these steps

### CODE REVIEW

**Comment by cooktheryan** [comment]
needs #10 first

File: ansible.cfg [hunk 1]
```diff
@@ -4,4 +4,3 @@
 forks = 50
 host_key_checking = False
 remote_user = ec2-user
-private_key_file = aws_keys_pairs.pem
```
Comment (sallyom) [inline ansible.cfg]: user will need to add that in, right? 
should it be 
`private_key_file = <enter private ssh key>` ?

Comment (cooktheryan) [reply ansible.cfg]: @sallyom so with this we should just put a note in that if you are using a key that is not the default ssh key for your system define it here. Also, we probably need to hardcode it in to the ansible-playbook step for the terraform steps

**Review by sallyom — APPROVED** [review]
LGTM!

**Pull request merged**